website/content/docs/connect/gateways/api-gateway/configuration/http-route.mdx

```diff
@@ -0,0 +1,605 @@
+requirements in an `http-route` configuration entry. Click on a property name
+to view additional details, including default values.
+
+- [`Kind`](#kind): string | must be `http-route`
+- [`Name`](#name): string | no default
+- [`Meta`](#meta): map | no default
+- [`Rules`](#rules): list | no default
+- [`Filters`](#rules-filters): map | no default
+  - [`Headers`](#rules-filters-headers): list | no default
+    - [`Add`](#rules-filters-headers): map | no default
+    - [`Remove`](#rules-filters-headers): list | no default
+    - [`Set`](#rules-filters-headers): map | no default
+  - [`URLRewrites`](#rules-filters-urlrewrites): list | no default
+    - [`Path`](#rules-urlrewrites-path): map | no default
+- [`Matches`](#rules-matches): list | no default
+  - [`Path`](#rules-matches-path): map | no default
+    - [`Match`](#rules-matches-path): string | no default
+    - [`Value`](#rules-matches-path): string | no default
+- [`Services`](#rules-services): list | no default
+  - [`Name`](#rules-services-name): string | no default
+  - [`Weight`](#rules-services-weight): number | `1`
+- [`Parents`](#parents): list | no default
+  - [`Kind`](#parents-kind): string | must be `api-gateway`
+  - [`Name`](#parents-name): string | no default
+  - [`SectionName`](#parents-sectionname): string | no default
+
+## Complete configuration
+
+When every field is defined, an `http-route` configuration entry has the following form:
+
+<CodeTabs>
+
+``` hcl
+Kind = "http-route"
+
+Name = "<name of the route>"
+
+Parents = [
+  {
+    Kind        = "api-gateway"
+    Name        = "<name of the api-gateway to bind to>"
+    SectionName = "<optional name of a specific listener on the api-gateway to bind to>"
+  }
+]
+
+Rules = [
+  {
+    Filters = {
+      Headers = [
+        {
+          Add = {
+            "<name of header to add>" = "<value of header to add>"
+          }
+          Remove = [
+            "<header to remove from request>"
+          ]
+          Set = {
+            "<name of header to set>" = "<value of header to set>"
+          }
+        }
+      ]
+      URLRewrite = {
+        Path = "<path to rewrite request to>"
+      }
+    }
+    Matches = [
+      {
+        Headers = [
+          {
+            Match = "<type of match: exact, prefix or regex>"
+            Name = "<name of header to match on>"
+            Value = "<value of header to match on>"
+          }
+        ]
+        Method = "<method type to match on>"
+        Path = {
+          Match = "<type of match: exact, prefix or regex>"
+          Value = "<value to match on>"
+        }
+        Query = [
+          {
+            Match = "<type of match: exact, present or regex>"
+            Name = "<name of query parameter to match on>"
+            Value = "<value of query parameter to match on>"
+          }
+        ]
+      }
+    ]
+    Services = [
+      {
+        Name   = "<name of Consul service to route to>"
```

Comment:
I'm assuming we typically don't document enterprise-specific fields here?

Reply:
@trujillo-adam what's the policy on that?

Reply:
Not much in terms of enterprise features, just the ability to target services/gateways within other namespaces/partitions